src/main/java/catan/domain/model/dashboard/NodeBean.java

```diff
@@ -126,6 +132,26 @@ public void setEdges(VerticalLinks<EdgeBean> edges) {
         this.edges = edges;
     }
 
+    public boolean noBuildingsOnNeighbourNodes() {
+        for (EdgeBean edge : getEdges().listAllNotNullItems()) {
+            for (NodeBean node : edge.getNodes().listAllNotNullItems()) {
+                if (!node.equals(this) && node.getBuilding() != null) {
+                    return false;
+                }
+            }
+        }
+        return true;
+    }
+
+    public boolean nearGameUsersNeighbourRoad(GameUserBean gameUser) {
```

Comment:
not consistnet names with previous method:
if you use both in the same statement,